packages/react/src/components/ZoomControl/index.tsx

```diff
@@ -44,6 +44,9 @@ const presets = [
   },
 ];
 
+const MAX_ZOOM_RATIO = 4;
+const MIN_ZOOM_RATIO = 0.1;
```

Comment:
It does not make any sense if you copy-paste them. You export and import.

Reply:
Do you mean for me to use a single if statement to handle the limiting process alone, just like you did in the `zoom.ts` file?

Reply:
Yeah, just check the range at the beginning of `zoomTo`